users/migrations/0001_initial.py

```diff
@@ -0,0 +1,67 @@
+# Generated by Django 4.0.3 on 2022-03-24 18:53
+
+import django.contrib.auth.models
+import django.contrib.auth.validators
+from django.db import migrations, models
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ('auth', '0012_alter_user_first_name_max_length'),
+    ]
+
```

Comment:
please add test user data
as you can see for example:
https://github.com/redhat-beyond/ClassRater/blob/main/homepage/migrations/0002_user_test_data.py

Reply:
Done